README.md

```diff
@@ -12,108 +59,157 @@ A REST API to support Reef Guide (AIMS), built with Express, TypeScript, Zod and
 - Environment-based configuration with Zod validation
 - Basic docker compose
 
-## Prerequisites
+## Quick Start
 
-- Node.js (v18+)
-- AWS CLI configured with appropriate permissions
-- Docker (for local development with Prisma)
+Get up and running with the API in minutes:
 
-## Setup (local)
+```bash
+# Clone the repository
+git clone <https://github.com/open-AIMS/reefguide-web-api.git>
+cd <reefguide-web-api>
 
-1. Clone the repository
-1. Install dependencies: `npm install`
-1. Generate Prisma client: `npm run prisma-generate`
-1. Set up environment variables: Copy `.env.example` to `.env`
-1. Generate JWT keys: `npm run local-keys`
-1. Start up psql `docker compose up`
-1. Run DB migration `npm run db-reset`
-1. Start server with auto-restart `npm run dev`
+# Install dependencies
+nvm use 20  # Or your preferred Node.js v18+ version
```

Comment:
I'm wondering why we're not using Node 22, but that's a question for Peter. Maybe due to AWS support?